server/main.lua

```diff
@@ -215,6 +219,32 @@ lib.callback.register('qbx_management:server:fireEmployee', function(source, emp
 	end
 end)
 
+-- Callback for giving a player a bonus from a given society.
+---@param employee string citizenid of employee to get the bonus
+---@param groupType GroupType
+---@param groupName string
+---@param amount number
+lib.callback.register('qbx_management:server:bonusEmployee', function(source, employee, groupType, groupName, amount)
```

Comment:
Not sure how I feel about adding back bank like activity to qbx_management. When we released it, we intentionally took out the bank features, intending this to be an employee management resource only; and bank stuff could be handled at the bank, or via another resource. This bonus feature seems to go against that original intention. 

I would pull it out into a separate PR to avoid blocking the time tracking feature, and I'll let some others weigh in with their thoughts.